### DISCUSSION TITLE
Sensors and values.

### DISCUSSION THREAD

**Discussion by JohanAlvedal** [springfall2008/batpred#709] [Q&A]
Hey all
I now realize that some of my sensors are faulty as the numbers are completely wrong
Could someone please explain which sensors it should be about kwh w daily toal etc. 
At the same time, which must and can be had.
As my inverter (huawei) is different from yours.

 Sensors, currently more than one can be specified and they will be summed up automatically
   however if you have two inverters only set one of them as they will both read the same.
  
 ```
 load_today:
    - sensor.solaredge_i2_ac_energy_kwh
  import_today:
    - sensor.solaredge_m1_imported_kwh
  export_today:
    - sensor.solaredge_m1_exported_kwh
  pv_today:
    - sensor.solar_panel_production_kwh   # This is a custom sensor - documentation TBD
  #
   Controls/status - must by 1 per inverter
  #
  num_inverters: 1
  inverter_type: "SE"

  battery_rate_max:
    - 11600
  charge_rate:
    - number.solaredge_i2_storage_charge_limit
  discharge_rate:
    - number.solaredge_i2_storage_discharge_limit
  #battery_power:
  #  - number.givtcp_{geserial}_battery_power
  #pv_power:
  #  - number.givtcp_{geserial}_pv_power
  #load_power:
  #  - number.givtcp_{geserial}_load_power
  soc_kw:
    - sensor.solaredge_b1_available_energy
  #soc_percent:
  #  - sensor.solaredge_b1_state_of_energy
  soc_max:
    - sensor.solaredge_b1_maximum_energy
  reserve:
    - number.solaredge_i2_backup_reserve
  #inverter_mode:
  # - select.givtcp_{geserial}_mode
  #inverter_time:
  # - sensor.givtcp_{geserial}_invertor_time
  charge_start_time:
   - "00:00:00"
  charge_end_time:
   - "00:01:00"
  charge_limit:
   - number.solaredge_i2_ac_charge_limit
  scheduled_charge_enable:
   - off
  scheduled_discharge_enable:
   - off
  discharge_start_time:
   - "00:00:00"
  discharge_end_time:
   - "00:01:00"
```

**Comment by JohanAlvedal** [comment]
There must be something fundamental I'm missing or the configuration is missing. Because I don't get any suggestions on when to charge the battery in or out

![IMG_1551](https://github.com/springfall2008/batpred/assets/25838439/e18f94c1-7dc7-46b5-a87e-f3d79797ee9b)

**Comment by springfall2008** [comment]
It seems like you are missing a load prediction, check the setting or load_today and check the log file for warnings

**Reply by JohanAlvedal** [reply]
I have a sensor there but it might be wrong
  load_today:
    - sensor.energy_today_kwh  
    - 
    - in kwh and daily but?

Can you explain which sensor it should be there.
from grid? the house's consumption
![predbat](https://github.com/springfall2008/batpred/assets/25838439/c5d9c000-eaa8-4ed7-b8df-5407b2a5fc30)

**Comment by springfall2008** [comment]
It’s meant to be in kWh and increasing, but resets are fine they are ignored (daily, monthly or forever). It’s meant to be house load - what you actually consume and not grid import.

maybe open the sensor and look at a history chart and see if it’s working

**Comment by springfall2008** [comment]
This maybe useful Predbat - Understanding Historical Data 📊
https://youtu.be/LrhDRUhziC4
